### PR TITLE
Post/two way databinding

### PR DESCRIPTION
**Pull request by javimsevilla**
Update libraries version and fix XML code samples. **Please check comments below.**

### CODE REVIEW

File: css/clean-blog.css [hunk 1]
```diff
@@ -409,190 +409,3 @@ footer .copyright {
   font-size: 16px;
   padding: 25px 35px;
 }
-
```
Comment (javimsevilla) [inline css/clean-blog.css:412]: Please check if this code is necessary. I removed it by accident when updating library versions

File: css/clean-blog.css [hunk 2]
```diff
@@ -409,190 +409,3 @@ footer .copyright {
...
-    height: 170px;
-    margin-left: -85px;
-  }
-  .timeline > li .timeline-image h4 {
-    margin-top: 40px;
-  }
-  .timeline > li.timeline-inverted > .timeline-panel {
-    padding: 0 100px 20px 20px;
-  }
-}
```
Comment (javimsevilla) [inline css/clean-blog.css:598]: Please check if this lines are necessary. I removed them by accident when I updated library versions

Comment (kikebodi) [reply css/clean-blog.css:598]: Is it possible to rebase that commit or add them in a new commit?

**Comment by kikebodi** [comment]
Cool! 

This fixes are for the base website so you can merge them to `master` instead of `post/two_way_databinding`

**Pull request merged**